upkie/cpp/actuation/BulletInterface.h

```diff
@@ -331,6 +331,12 @@ class BulletInterface : public Interface {
   //! Randomize masses of the robot links
   void randomize_masses();
 
+  //! Check if the robot collides with its environment
+  int environment_collision();
+
+  //! Is the robot colliding with its environment
+  int environment_collision_;
```

Comment:
The trailing underscore is a C++ convention to indicate that an attribute is private. Thus here either `environment_collision_` should be private (what you intended), or it should be named `environment_collision`.